### ISSUE TITLE
Allow full item description in dropdown field

### ISSUE THREAD

**Issue by raphattack** [inn3rvoice/inventory_alpha#6]
This is an enhancement request.

Currently the item affix/description is being cut off in the dropdown field:
![image](https://github.com/inn3rvoice/inventory_alpha/assets/1328300/405b0b8f-a94e-4928-bc66-6f804e617a58)

Screenshot is from a 3440x1440 resolution.

This request is to modify the dropdown field so that the full item affix/description can be seen.

**Comment by dragonalvaro** [comment]
Agree, rogue has two aspects which starts very similar and the tool thinks that are the same aspect.

**Comment by dragonalvaro** [comment]
In druid, Stormclaw, Conduit and Accelerating, al three starts with: "Critical strikes with..." and the overlay merges the three of them in the same category

![image](https://github.com/inn3rvoice/inventory_alpha/assets/4569309/7785f6f9-54b8-4eba-80ea-c7f5d0decf39)

Descriptions should be longer for items

**Comment by dragonalvaro** [comment]
![image](https://github.com/inn3rvoice/inventory_alpha/assets/4569309/cc59a97c-75aa-4cd8-bd4c-03b84f778d64)
![image](https://github.com/inn3rvoice/inventory_alpha/assets/4569309/5062252d-5d45-41f3-bdb6-3f668ddc3593)
![image](https://github.com/inn3rvoice/inventory_alpha/assets/4569309/0feee4c7-865a-48db-bef2-817c30658c44)
